packages/eslint-plugin/src/rules/no-deprecated-btn-rg.ts

```diff
@@ -0,0 +1,38 @@
+import { createRule } from '../utils/create-rule';
+import { TemplateNode } from '../template-parser/template-node';
+
+export const name = 'no-deprecated-btn-rg';
+
+// Type: RuleModule<"uppercase", ...>
+export default createRule({
+  name,
+  meta: {
+    docs: {
+      category: 'template',
+      description:
+        'Flags deprecated "btn-rg" class and suggests removal or replacement with "btn-sm".',
+    },
+    messages: {
+      deprecatedBtnRg:
+        'The "btn-rg" class is deprecated. Please remove it or replace it with "btn-sm".',
+    },
+    type: 'suggestion',
+    schema: [],
+  },
+  defaultOptions: [],
+  create(context) {
+    return {
+      tag(node: TemplateNode) {
+        if (node.name === 'button') {
+          const $node = node.toCheerio();
+          if ($node.hasClass('btn-rg')) {
+            context.report({
```

Comment:
Is it possible to auto-fix instead of just report?

Reply:
Sure! But for this rule I am unsure what to do: replace the class with `.btn-sm`? remove the class altogether? I therefore removed the fixer to let project decide case by case. I can add it back if you have a preferred solution.

Reply:
I think it would be useful to have fixes for most of the things. Even if it does not map 100%, if we can handle 80% of the cases automatically, that will be a huge boost compared to have to do 100% of the changes manually.

For this specific case, `btn-sm` seems to be closer to what `btn-rg` was (40px on Desktop), therefore I'd probably use this instead of removing it, which would fall back to medium.